chart/k8gb/values.yaml

```diff
@@ -11,6 +11,7 @@ k8gb:
   # but can be overrided with imageTag key
   # imageTag:
   dnsZone: "cloud.example.com" # dnsZone controlled by gslb
+  dnsZoneNegTTL: 3600 # Negative TTL for SOA record
```

Comment:
why default negative is 3600 when it is clearly problematic got GSLB scenario? 
```suggestion
  dnsZoneNegTTL: 60 # Negative TTL for SOA record
```

Reply:
What about we don't set any default value and keep plugins one, if set it will be propogated

Reply:
@k0da i'm concerned that another customers will hit the same issue without having a clue that the root cause is actually negative TTL.  It is safer to keep it low imho

Reply:
10 minutes?

Reply:
why?

Reply:
IMO 1 minute is too agressive

Reply:
agreed on 5 minutes